src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -249,6 +249,43 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
     error. This is to throw a 404 when an asset, or some other file is not
     found.
 
+    ## Routing prefix
+
+    More common with APIs, sometimes you need a group of routes to be prefixed with some path.
```

Comment:
```suggestion
    Sometimes you need a group of routes to be prefixed with some path.
```

I think the next sentence shows an example of an API so I think it can be left off the first sentence